Competition2017/src/competition/subsystems/shooter_belt/ShooterBeltSubsystem.java

```diff
@@ -33,8 +35,10 @@ public ShooterBeltSubsystem(
         this.side = side;
         
         intakePowerProperty = propManager.createPersistentProperty("ShooterBelt intake power", 0.5);
+        intakeTracerPowerProperty = propManager.createPersistentProperty("ShooterBelt intake tracer power", 0.25);
```

Comment:
It's unclear what these are. Are the two properties supposed to be equivalent values for running the belt in either speed or power mode?